src/components/ArticleController.jsx

```diff
@@ -19,17 +19,15 @@ export default class ArticleController extends FalcorController {
       let articleSlug = this.props.params.articleSlug;
       // Access data fetched via Falcor
       const articleData = this.state.data.articlesBySlug[articleSlug];
-      //console.log("Data: " + JSON.stringify(articleData));
       return (
         <div>
-          {/*
-            <div>Controller for article: {articleData.title}</div>
-            <div>Ready?: {this.state.ready ? 'true' : 'false'}</div>
-          */}
           <Article
             title={articleData.title}
+            teaser={articleData.teaser}
+            pubDate={articleData.published_at}
             html={articleData.html}
             authors={articleData.authors}
+            url={"thegazelle.org/issue/" + articleData.issueId + '/' + articleData.category + '/' + articleData.slug}
```

Comment:
We won't get the benefit of ghost meta tag rendering because we aren't
letting ghost render our pages. It's another TO-DO for us.

On Mon, Aug 8, 2016, 6:43 AM Zane Mountcastle <notifications@github.com>
wrote:

> In src/components/ArticleController.jsx
> <https://github.com/thegazelle-ad/gazelle-front-end/pull/41#discussion_r73821655>
> :
>
> >              html={articleData.html}
> >              authors={articleData.authors}
> > +            url={"thegazelle.org/issue/" + articleData.issueId + '/' + articleData.category + '/' + articleData.slug}
>
> I actually just tested the facebook issue with an old article and it works
> both ways -- with or without https://www.
>
> —
> You are receiving this because you were assigned.
>
>
> Reply to this email directly, view it on GitHub
> <https://github.com/thegazelle-ad/gazelle-front-end/pull/41/files/a85a19c83b836058745a11879e25ef14a8a7a67e#r73821655>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ADAFzVT_XkL1fpEEoM8vCy-q0Qa3U98aks5qdrP7gaJpZM4JdXZj>
> .
>